docs-site/src/css/responsive.css

```
@@ -0,0 +1,189 @@
+  }
+
+  .docItemContainer_src-theme-DocItem-Layout-styles-module,
+  [class*="docItemContainer"] {
+    padding: 0 var(--spacing-2);
+  }
+
+  /* Prevent content from exceeding viewport */
+  .theme-doc-markdown,
+  .markdown {
+    overflow-wrap: break-word;
+    word-break: break-word;
```

Stylelint (17.3.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)